demo/info_section.js

```diff
@@ -196,11 +195,11 @@ shakaDemo.updateLanguageOptions_ =
   var selectedTrack = activeTracks[0];
 
   // Populate list with new options.
-  languages.forEach(function(lang) {
+  tracks.forEach(function(track) {
```

Comment:
Tests are failing, so I will leave it to you to make updates.

For the demo app, please revert changes in this file and look at the `formatters` map in `updateTrackOptions_`.  That is where you should add labels to the track list.  Here's a suggestion:

```js
shakaDemo.updateTrackOptions_ = function(list, tracks, language) {
  var formatters = {
    variant: function(track) {
      var trackInfo = '';
      if (track.language) trackInfo += 'language: ' + track.language + ', ';
      if (track.label) trackInfo += 'label: ' + track.label + ', ';
      if (track.width && track.height)
        trackInfo += track.width + 'x' + track.height + ', ';
      trackInfo += track.bandwidth + ' bits/s';
      return trackInfo;
    },
    text: function(track) {
      var trackInfo = 'language: ' + track.language + ', ';
      if (track.label) trackInfo += 'label: ' + track.label + ', ';
      trackInfo += 'kind: ' + track.kind;
      return trackInfo;
    }
  };
```